### DISCUSSION TITLE
Not fully uninstalled / causing crashes?

### DISCUSSION THREAD

**Discussion by NoEyeSpec** [ChromaControl/ChromaControl#225] [General]
Sorry all I have tried to join the discord to ask this question but despite accepting the server rules I am still locked out of making any posts whatsoever.

I am having an issue with a game (MW2), a recent update has caused the game to start crashing upon loading.  The crash reports on the game logs show this information:

------------------------
Windows Error Reporting:
+++ WER0 +++:
Fault bucket 1566938115028375692, type 5
Event Name: MoAppCrash
Response: Not available
Cab Id: 0

Problem signature:
P1: ChromaControl_0.1.3.0_x64__q2sx6sw85pbp2
P2: praid:App
P3: 0.1.3.0
P4: 60c8ffcd
P5: KERNELBASE.dll
P6: 10.0.22621.1194
P7: faa44dd0
P8: e0434352
P9: 000000000009051c
P10: 
------------------------------

ChromaControl is showing up as causing a problem, but I do not still have it installed on my system.  I had it for a brief time but wasn't using it so I uninstalled it.  It does not show up on the old style windows uninstaller nor does it show up in the new settings -> apps in W11.

Any clue at all as to what could be going on or where the app is still located on my system and how I can remove it completely?

Thanks for any help you can offer.

**Comment by andrewbabbittdev** [comment]
It's windows responsibility to remove any lingering files, as windows manages the lifecycle of the install process.

If you uninstalled the app windows should have removed all of the program files from your PC. Does any Chroma Control executable show up in the task manager? If not then this is a real head scratcher.
